src/sales/credit-memo-create.md

```diff
@@ -156,18 +174,16 @@ Determine the type of refund that you want to issue. Then, use one of the follow
       The status of a credit memo notification appears in the completed credit memo next to the credit memo number.
 
       ![]({% link images/images-b2b/credit-memo-order-totals.png %}){: .zoom}
-      _Refund Totals_
+      _Refund Totals_{:.b2b-only}
```

Comment:
We are taking the edition tags off of the captions because it was too confusing for people. If the accompanying information calls out things that are edition specific, that is preferable.
```suggestion
      _Refund Totals_
```